sqlglot/dialects/spark.py

```diff
@@ -44,6 +46,14 @@ def _unix_to_time(self, expression):
     raise ValueError("Improper scale for timestamp")
 
 
+def _timestamp_trunc(args: t.Sequence) -> exp.Expression:
+    unit = seq_get(args, 0)
+    if isinstance(unit, exp.Column):
```

Comment:
why would this happen? there's no test case for this, can you just use rename_func?